packages/popover/src/Popover.types.ts

```diff
@@ -11,6 +11,39 @@ type TransitionLifecycleCallbacks = Pick<
   'onEnter' | 'onEntering' | 'onEntered' | 'onExit' | 'onExiting' | 'onExited'
 >;
 
+/**
+ * Options to render the popover element
+ * @param Inline will render the popover element inline with the reference element
```

Comment:
`inline` can have different meanings. Can we be clear that we mean "in the DOM where it's written" and not "display: inline`